.github/workflows/build.yaml

```diff
@@ -92,9 +92,11 @@ jobs:
         uses: ludeeus/action-shellcheck@master
 
   docker-build-check:
-    name: Docker image build check
+    name: “Check whether docker single-node and cluster workflows build correctly.”
```

Comment:
Very unusual quotes here

Reply:
I noticed that too, Emacs thinking YAML should have smartquotes....waiting to push the patch to avoid interrupting CI.